astropy/visualization/wcsaxes/core.py

```diff
@@ -746,6 +746,44 @@ def tick_params(self, axis='both', **kwargs):
                 if spine in coord.axislabels.get_visible_axes():
                     coord.tick_params(**kwargs)
 
+    def set_world_aspect(self, aspect, **kwargs):
+        """
+        Set the aspect of the axes in world coordinates.
+
+        Parameters
+        ----------
+        aspect : float
+             A circle will be stretched such that the height is *aspect*
```

Comment:
A circle in world coordinates or pixel coordinates? What does 'circle' mean if the axes are not both celestial?